desktop/core/src/desktop/api_public_urls.py

```diff
@@ -75,6 +75,6 @@
 ]
 
 # Slack install API for using CORS by default
-urlpatterns = [
+urlpatterns += [
```

Comment:
Sorry for this mistake @romainr

Reply:
No worries, good lesson to add more tests!